lib/server/controllers/logs.controller.js

```diff
@@ -281,6 +283,40 @@ const getLogById = async (request, response, next) => {
     }
 };
 
+/**
+ * Endpoint to list all the root logs
+ *
+ * @param {Object} request the express request
+ * @param {Object} response the express response
+ * @return {void}
+ */
+const getRootLogs = async (request, response) => {
+    const value = await dtoValidator(GetAllLogsDto, request, response);
```

Comment:
In your case, you don't have parameters such as filter that you may find in GetAllLogsDto. You should build a custom DTO to simply look for pagination.
You can find an example here: https://github.com/AliceO2Group/Bookkeeping/blob/9114239e18201fbb83fc9dfc513aef6144c3a87e/lib/server/controllers/dataPasses.controller.js
By removing the filtering option too.

Reply:
done

Reply:
done